.gitlab-ci/stages/notify.yml

```diff
@@ -61,6 +66,26 @@ Release:
     - git config user.email ${CI_GIT_AUTHOR_EMAIL}
     - git remote set-url origin https://${GITHUB_TOKEN}@github.com/${CI_PROJECT_PATH}.git
   script:
-    - yarn --frozen-lockfile
     - GH_TOKEN=${GITHUB_TOKEN} yarn lerna version ${LERNA_ARGS:="--yes"}
 
+#
+#
+#
+
+Production:
+  <<: *notify_stage
+  stage: "Notify Finished Deployment"
+  image: $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA
```

Comment:
we actually just need `curl` here

Reply:
yes indeed ! we coud use a lighter image